src/app/app.component.html

```diff
@@ -2,16 +2,10 @@
 <html lang="en">
 
 <head>
-  <title>{{'APP.TITLE' | translate}}</title>
+  <title>{{'APP.TITLE'}}</title>
```

Comment:
in this html file we should only have... nothing at this point, your full html structure is in index.html where you have a html tag title tag and so on , here we will have some custom html for app component in the future, I will make a changes to this file and explain you this during our call